lms/urls.py

```diff
@@ -20,9 +20,7 @@
 from openedx.features.enterprise_support.api import enterprise_enabled
 
 
-# Uncomment the next two lines to enable the admin:
-if settings.DEBUG or settings.FEATURES.get('ENABLE_DJANGO_ADMIN_SITE'):
-    admin.autodiscover()
```

Comment:
Why was the feature flag check taken out? Will the feature flag be removed as well?